src/wui/load_or_save_game.cc

```diff
@@ -135,39 +122,41 @@ bool LoadOrSaveGame::compare_map_name(uint32_t rowa, uint32_t rowb) const {
 std::unique_ptr<SavegameData> LoadOrSaveGame::entry_selected() {
 	std::unique_ptr<SavegameData> result(new SavegameData());
 	size_t selections = table_->selections().size();
-	if (selections == 1) {
+	set_tooltips_of_buttons(selections);
+
+	const std::vector<SavegameData> savegames = get_selected_savegames();
+	game_details_.display(savegames);
+	if (selections > 0 && !selection_contains_directory()) {
```

Comment:
```suggestion
	if (!selections.empty() && !selection_contains_directory()) {
```

Reply:
??
selections is of type size_t so it does not have .empty()

Reply:
Oops, I have updated the code suggestion. You can probably get rid of the selections variable then.

Reply:
yeah, did that